File: Ex2.2/ex2.2.py
def le():
    num1=int(raw_input("digite um numero: "))
    num2=int(raw_input("digite um numero: "))
    return num1 , num2
def impr(num1,num2):
    for i in range(num1+1,num2):
        print i 
    return
#main
num1,num2=le()
impr(num1,num2)

File: Ex2.2/ex2.2.c
#include <stdio.h>
int main(void){
	int num1;
	int num2;
	printf("digite um numero: ");
	scanf("%d", &num1);
	printf("digite outro numero: ");
	scanf("%d", &num2);
	if(num1<num2){
		while(num1 < (num2-1)){
			num1=num1+1;
			printf("%d\n", num1);
		}
	}else if(num2<num1){
		while(num2<(num1-1)){
			num2=num2+1;
			printf("%d\n", num2);
		}
	}else{
		printf("0");
	}
return 0;
}
